.travis.yml

```diff
@@ -20,10 +20,10 @@ install:
   - conda create --yes -q -n pyenv python=$TRAVIS_PYTHON_VERSION numpy=1.9.2 scipy=0.16.0 nose=1.3.7 six sphinx=1.3
   - source activate pyenv
   - case "$TRAVIS_PYTHON_VERSION" in 2.*) SPECIAL_PACKAGES="mock";; 3.*) SPECIAL_PACKAGES="";; esac
-  - pip install  coveralls $SPECIAL_PACKAGES
+  - pip install  coveralls tempdir $SPECIAL_PACKAGES
```

Comment:
you might want to add `six` as well here. I think I've used to be py2/[y3 compatible.